sharding/nodesSetup_test.go

```diff
@@ -14,7 +14,8 @@ func createNodesSetupOneShardOneNode() *sharding.NodesSetup {
 	ns.MinNodesPerShard = 1
 	ns.InitialNodes = make([]*sharding.InitialNode, 1)
 	ns.InitialNodes[0] = &sharding.InitialNode{}
-	ns.InitialNodes[0].PubKey = "5126b6505a73e59a994caa8f556f8c335d4399229de42102bb4814ca261c7419"
+	ns.InitialNodes[0].PubKey = "52f3bf5c01771f601ec2137e267319ab6716ef6ff5dfddaea48b42d955f631167f2ce19296a202bb8fd174f4e94f8c85f619df85a7f9f8de0f3768e5e6d8c48187b767deccf9829be246aa331aa86d182eb8fa28ea8a3e45d357ed1647a9be020a5569d686253a6f89e9123c7f21f302e82f67d3e3cd69cf267b9910a663ef32"
```

Comment:
can we add these pubKeys and addresses as constants in the file, maybe in an array testPubKeys and testAddresses and use the constants instead for initialization in tests as it doesn't read too well. same for the other tests in this file that make use of these.

Reply:
+1

Reply:
Refactored

Reply:
+2 :)